_posts/2021-11-23-release-2.26.0.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: default
+title: Release v2.26.0
+tags: ["v2.26.0", "changelog"]
+excerpt_separator: <!--more-->
+---
+
+As of 23/11 2021 the new version 2.26.0 is out
+<!--more-->
+
+## Menas Improvements
+
+- [#1938]({{ site.github.issues_url }}/1938) Mapping Conformance Rule may now exist without a join conditions, but in such a case a non-empty filter must be defined instead.
+- [#1904]({{ site.github.issues_url }}/1904) On Import Dataset, all output columns of MappingTableConformanceRule will be validated
+- [#1954]({{ site.github.issues_url }}/1954) When importing a dataset a mapping table conformance rule can now output a struct and will be properly validated.
+
+## New Standardization and Conformance Improvements and Changes
+
+- [#1968]({{ site.github.issues_url }}/1968) Releasing cached standardization data after its phase in the combined _Standardization&Conformance_ job, reducing its memory requirement.
+- [#1867]({{ site.github.issues_url }}/1867) Enceladus no longer support loading filter from config (config tree `dataframefilter.$dataFrameName`).
+
+## HyperConformance Plugin Improvements
+
+- [#1898]({{ site.github.issues_url }}/1898) `enceladus_info_version` can now be taken from a conformed column in HyperConformance
+
+## Menas Fixes
+
+- [#1966]({{ site.github.issues_url }}/1966) Add missing "Add Control Checkpoint" checkbox in Mapping Table Conformance Rule
+- [#1978]({{ site.github.issues_url }}/1978) Filter edit icons are now switched to better convey their expected meaning and to conform to the icons used in the filter view.
+- [#1981]({{ site.github.issues_url }}/1981) Bugfix: The _Conformance Rules_ view now correctly displays an indexed label and _Check Control Measures_ checkbox view for MappingConformanceRules (both missing since v2.22.0)
```

Comment:
```suggestion
- [#1966]({{ site.github.issues_url }}/1966) Add missing "Add Control Checkpoint" checkbox in Mapping Table Conformance Rule.
- [#1978]({{ site.github.issues_url }}/1978) Filter edit icons are now switched to better convey their expected meaning and to conform to the icons used in the filter view.
- [#1981]({{ site.github.issues_url }}/1981) Bugfix: The _Conformance Rules_ view now correctly displays an indexed label and _Check Control Measures_ checkbox view for MappingConformanceRules (both missing since v2.22.0).
```